torch/_C/__init__.pyi.in

```diff
@@ -49,6 +50,8 @@ from torch.types import (
     SymInt,
 )
 
+from numpy._typing import _NestedSequence
```

Comment:
sus. Do we really want to import numpy here?

Reply:
Yes, I wasn't sure about this one since it's a private member. Would it be ok to simply duplicate the protocol?

https://github.com/numpy/numpy/blob/7d4acf9afdea6871695586c868d6db6ba772b995/numpy/_typing/_nested_sequence.py#L19

Reply:
Let's dupe the protocol